R/LSBuffonsneedlemanipulation.R

```diff
@@ -96,7 +95,30 @@ LSBuffonsneedlemanipulation   <- function(jaspResults, dataset, options, state =
     
     if (options[["legendPropDistPlot"]]){
       propDistPlot$plotObject <-  propDistPlot$plotObject + 
-        ggplot2::theme(legend.position = c(.17, .9))
+        ggplot2::theme(legend.position = "right")
+    }
+    
+    if (options[["CIPropDistPlot"]]){
+      propCI95lower <- qbeta((1-options[["CI"]])/2, options[["k"]], options[["n"]] - options[["k"]], lower.tail = FALSE) 
+      propCI95lower <- round(propCI95lower, digit = 2)
+      
+      propmed <- qbeta(.5, options[["k"]], options[["n"]] - options[["k"]], lower.tail = FALSE)
+      propmed <- round(propmed, digit = 2)
+      
+      propCI95upper <- qbeta(1-(1-options[["CI"]])/2, options[["k"]], options[["n"]] - options[["k"]], lower.tail = FALSE)
+      propCI95upper <- round(propCI95upper, digit = 2)
+      
+      propDistPlot$plotObject <- propDistPlot$plotObject +
+        ggplot2::annotate("text", x = 0.75, y = 1.6*max(propPost), 
+                          label = gettextf("%s%% CI: [%s, %s]", options[["CI"]]*100, propCI95lower, propCI95upper),
```

Comment:
```suggestion
                          label = gettextf("%1$s%% CI: [%2$s, %2$s]", options[["CI"]]*100, propCI95lower, propCI95upper),
```

here we need the positional arguments to improve translatability (see https://github.com/jasp-stats/jasp-desktop/pull/4883)

Reply:
should be `gettextf("%1$s%% CI: [%2$s, %3$s]"`

Reply:
yeah, number is the position order of arguments